src/status_im/chat/events/commands.cljs

```diff
@@ -39,7 +39,8 @@
             to          (get-in contacts [chat-id :address])
             jail-params {:parameters params
                          :context    (generate-context current-account-id chat-id to group-id)}]
-        {:call-jail {:jail-id                 jail-id
+        {:db        db
```

Comment:
What's the reason for including `:db` effect here ?

Reply:
because we modify it at an earlier stage and we need these modifications to be there

Reply:
Where is that ? I fail to see it in the function or in any place where function is called.

Reply:
`status-im.chat.events.input/command-complete-fx` -> `status-im.chat.events.input/proceed-command` -> `status-im.chat.events.input/request-command-data` -> `status-im.chat.events.commands/request-command-message-data`